src/bench/addrman.cpp

```diff
@@ -109,9 +109,10 @@ static void AddrManGetAddr(benchmark::Bench& bench)
 static void AddrManAddThenGood(benchmark::Bench& bench)
 {
     auto markSomeAsGood = [](AddrMan& addrman) {
+        const auto now{AdjustedTime()};
         for (size_t source_i = 0; source_i < NUM_SOURCES; ++source_i) {
             for (size_t addr_i = 0; addr_i < NUM_ADDRESSES_PER_SOURCE; ++addr_i) {
-                addrman.Good(g_addresses[source_i][addr_i]);
+                addrman.Good(g_addresses[source_i][addr_i], now);
```

Comment:
I didn't like the implicit logic, relying on a "global" (the time), so I removed the default arg. But I am happy to drop the commit.

Reply:
I had the same intuition -- had to look up the standard to check AdjustedTime() would even be called each time rather than just once at startup -- but the refactor makes it a lot more wordy, so it just doesn't seem like a win to me *shrug*

Reply:
Removed commit for now.